index.html

```diff
@@ -294,13 +285,15 @@
         <div class="links-mobile">
             <ul class="icons">
                 <li><a href=#"><img src="./images/twitter.png"></a></li>
-                <li><a href=#"><img src="./images/i.png"><img src="./images/n.png"></a></li>
+                <li><a href=#"><img src="./images/in.png"></a></li>
                 <li><a href=#"><img src="./images/m.png"></a></li>
                 <li><a href=#"><img src="./images/github.png"></a></li>
                 <li><a href=#"><img src="./images/v.png"></a></li>
-                <p>ehmaddd@gmail.com</p>
             </ul>
         </div>     
     </section>
+    <div class="footer">
+        <p>ehmaddd@gmail.com</p>
+    </div>
 </body>
 </html>
```

Comment:
- [ ] Please, kindly fix the `overflow-x` anomaly, to enable you to have a very beautiful design,

![mobile](https://user-images.githubusercontent.com/41163571/220490586-541c16fb-958c-4275-8340-0eb47ee3bc66.gif)

Reply:
![image](https://user-images.githubusercontent.com/54537654/220784114-5e4e28e3-d754-4d78-b08a-e0e4645229dd.png)
Now you will notice that everything is properly aligned from top to bottom and nothing will show such anamoly.